backend/tests/unit/dal/Libraries/Ches/ChesExceptionTest.cs

```diff
@@ -0,0 +1,149 @@
+using FluentAssertions;
+using Pims.Core.Exceptions;
+using Pims.Core.Http;
+using Pims.Core.Test;
+using Pims.Ches.Models;
+using Moq;
+using System.Diagnostics.CodeAnalysis;
+using System.Net;
+using Xunit;
+using Pims.Core.Http.Models;
+using Microsoft.Extensions.Options;
+using Pims.Ches.Configuration;
+using Pims.Ches;
+using System.Linq;
+using System.Net.Http;
+
+namespace Pims.Dal.Test.Libraries.Ches
+{
+    [Trait("category", "unit")]
+    [Trait("category", "ches")]
+    [Trait("group", "notification")]
+    [ExcludeFromCodeCoverage]
+    public class ChesExceptionTest
+    {
+
+        #region Tests
+        #region ChesException Constructors
+        [Fact]
+        public void ChesExceptionClientModel_Success()
+        {
+            // Arrange
+            var message = "test2";
+            var status = HttpStatusCode.OK;
+            var helper = new TestHelper();
+
+            var user = PrincipalHelper.CreateForPermission();
+
+            var options = Options.Create(new ChesOptions()
+            {
+                AuthUrl = "https:/test.com",
+                EmailEnabled = true,
+                EmailAuthorized = true,
+                BccUser = true
+            });
+            var service = helper.Create<ChesService>(options, user);
+
+            var token = new TokenModel()
+            {
+                AccessToken = "test"
+            };
+
+            var exception = new HttpClientRequestException(message, status);
+            var client = helper.GetService<Mock<IHttpRequestClient>>().Object;
+            var model = new ErrorResponseModel();
+
+            // Act
+            var chesException = new ChesException(exception, client, model);
+
+            // Assert
+            Assert.NotNull(exception);
+            Assert.NotNull(client);
+            Assert.NotNull(model);
```

Comment:
No need to assert a variable you created.

Reply:
Should I remove those 3 assertions, or can we assert the chesException properties as below?

`Assert.NotNull(chesException.Message);
 Assert.NotNull(chesException.Client);
 Assert.NotNull(chesException.Detail);`

Reply:
Remove the two assertions that are related to the `Act`.  You will always know `client` and `model` are not null because you created them yourself manually.